Bor/Program.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Bor;
+
+internal class Program
+{
+    static void Main()
+    {
+        if (!Test.TestBor())
+        {
+            Console.WriteLine("Error!");
+        }
+    }
+}
```

Comment:
```suggestion
if (!Test.TestBor())
{
       Console.WriteLine("Error!");
}
```
И хорошо бы что-то печатать при успехе.